InnerEye/ML/utils/supervised_criterion.py

```diff
@@ -74,15 +75,13 @@ def get_positive_class_weights(self) -> torch.Tensor:
         :return: a list of weights to use for the positive class for each target position.
         """
         assert self._class_counts is not None
-        labels = list(self._class_counts.keys())
-        if sorted(labels) != [0.0, 1.0]:
-            if labels == [1.0] or labels == [0.0]:
-                return torch.tensor(1.0)
-            else:
-                raise ValueError(f"Expected one-hot encoded binary label."
-                                 f"Found labels {self._class_counts.keys()}")
-        else:
+        if self.num_classes == 1:
             return torch.tensor(float(self._class_counts[0.0]) / self._class_counts[1.0], dtype=torch.float32)
```

Comment:
Why is this (# of Class 0 samples) / (# of Class 1 samples) in the binary case and (# of Class 'i' samples) / (total # of samples) in the multilabel case? It should be consistent.